Tasks/UseNodeV1/proxyutil.ts

```diff
@@ -0,0 +1,44 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as qs from 'querystring';
+import * as url from 'url';
+
+interface ICurlProxy {
+    variable: string,
+    setting: string
+}
+
+function toCurlProxy(proxyCfg: taskLib.ProxyConfiguration): ICurlProxy {
```

Comment:
why wouldnt the agent just set this? this isnt a "node" thing, this is a linux thing. am i right?

Reply:
I think we want to leave this here - you could argue we want to set this by default for Linux agents, but even if we do that I think we still need this here for Windows agents (and I don't think we should set it by default for Windows agents).

@bryanmacfarlane any thoughts?